src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3150,6 +3150,9 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Fixing_0" xml:space="preserve">
     <value>Fixing '{0}'</value>
   </data>
+  <data name="Insert_a_for_loop" xml:space="preserve">
+    <value>Insert a 'for' loop</value>
+  </data>
```

Comment:
Will the same string be used for VB in future?

If not, consider moving to `CSharpFeaturesResources`

Reply:
Good point